features/frameworks/greentea-client/greentea-client/greentea_metrics.h

```diff
@@ -7,11 +26,13 @@
 /**
  *  Setup platform specific metrics
  */
+MBED_DEPRECATED_SINCE("mbed-os-6.13", "Greentea metrics API are deprecated")
```

Comment:
We released mbed-os-6.13 today, so the starting version will be mbed-os-6.14

Reply:
Correct, lets fix it